### DISCUSSION TITLE
how to use qsv to xlsx in a pipeline

### DISCUSSION THREAD

**Discussion by chwwong** [dathere/qsv#907] [Q&A] [answered]
when using qsv to   as the target of a pipeline of qsv commands, does not allow pipe?
eg

`qsv select 1,4 testdata.csv | qsv to xlsx test.xlsx`

fails with "Need to add the name of a xlsx file as first argument then the input CSVs" error

It only works if you specify the csv file on the command line

**Comment by jqnatividad** [comment]
Hi @chwwong ,

I'm afraid, the `to` command currently does not support stdin.

Will add it to the backlog.

cc @kindly

**Comment by jqnatividad** [comment, marked as answer]
Hi @chwwong,

I'm glad to report that with https://github.com/jqnatividad/qsv/pull/913 and the 0.99.0 release, the `to` command now supports stdin.
